scrypt.1

```diff
@@ -113,11 +108,8 @@ possible value for
 .Ar maxmemfrac
 is 0.5.
 .It Fl P
-Always read passphrase from stdin, and do so only once even
-when encrypting.
-This cannot be used if
-.Ar infile
-is also stdin (aka '-').
+Do the same thing as
```

Comment:
I would write this as "Deprecated synonym for"

Reply:
Fixed as a REBASE commit.  If you're finished looking at this PR, I'll rebase and re-upload.